### PR TITLE
CORE-8194: Define the producers and consumers of flow topics

### PR DESCRIPTION
**Pull request by fowlerrr**

### CODE REVIEW

File: data/topic-schema/src/main/resources/net/corda/schema/Flow.yaml [hunk 1]
```diff
@@ -18,17 +25,24 @@ topics:
...
     producers:
+      - flow
     config:
   FlowMapperEventTopic:
     name: flow.mapper.event
     consumers:
+      - flow
     producers:
+      - flow
+      - rpc
```
Comment (mnesbit) [inline data/topic-schema/src/main/resources/net/corda/schema/Flow.yaml:38]: I guess p2p inbound is fetched via a process in flow processor so this is correct

**Comment by mnesbit** [comment]
make sense to me

**Comment by fowlerrr** [comment]
Looks to be passing the e2e tests: https://ci02.dev.r3.com/job/Corda5/job/corda-runtime-os-e2e-tests/job/PR-2734/2/

**Review by davidcurrie — APPROVED** [review]
👍

**Review by JamesHR3 — APPROVED** [review]

**Pull request merged**